### DISCUSSION TITLE
Hey i wanna know how to add player options like `leaveOnEmpty: true/false` as i didn't found any example in the discord-player docs.

### DISCUSSION THREAD

**Discussion by BEAST2802** [Androz2091/discord-player#715] [Q&A]
Hey i wanna know how to add player options like `leaveOnEmpty: true/false` as i didn't found any example in the discord-player docs.

_Originally posted by @BeastBoyADI in https://github.com/Androz2091/discord-player/discussions/713_

**Comment by BEAST2802** [comment]
And yeah i wanted to make my bot stay in vc for 24/7 so need to know how to add player options so that i can use it for making my bot 24/7 in vc

**Comment by twlite** [comment]
`leaveOnEmpty` should be passed to Queue while creating a queue. For 24/7, you need to disable `leaveOnEmpty`, `leaveOnEnd` and then whenever a track/queue ends, you need to queue up something. (for example: youtube autoplay mode)

**Comment by BEAST2802** [comment]
so can u give an example like how do i pass it while creating the queue using `.createQueue()` as i am a lil bit confused how to put it inside that to make it work

**Reply by marcosebsilva** [reply]
I know this is a little outdated but **leaveOnEnd** should be a key in the options object that you pass to createQueue.

 ```
const queue = player.createQueue(interaction.guild, {
          metadata: `{`
              channel: interaction.channel,
          },
          leaveOnEmpty: false, // dont leave when the channel is empty
          leaveOnEnd: false, // dont leave when the queue ends
      });
```

**Comment by BEAST2802** [comment]
Like for an example i want to just disable `leaveOnEmpty`. So can u give an example by putting `leaveOnEmpty: false` in `.createQueue()` so i can get to know the correct format

**Comment by BEAST2802** [comment]
Hello u there? Please give an example so that i can get to know the correct format

**Reply by milbits** [reply]
just read the docs damnit

**Reply by TheMonDon** [reply]
> just read the docs damnit

The docs are not updated to v5 very well. There is nothing saying where to put this.
